React-Ex3/03-reactapp/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ReactDOM from "react";
+import "./App.css";
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+
+/*
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
+*/
+
+function WelcomePage() {
+  return (
+    <Navbar expand="lg" className="bg-body-tertiary">
+      <Container>
+        <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="me-auto">
+            <Nav.Link href="#home">Home</Nav.Link>
+            <Nav.Link href="#search">Search</Nav.Link>
+            <Nav.Link href="#">Houses</Nav.Link>
+          </Nav>
+        </Navbar.Collapse>
+      </Container>
+    </Navbar>
+  );
+}
+
+export default WelcomePage;
+const welcome = React.createElement("h1", "Welcome!");
+React.DOM.render(welcome, document.getElementById("root"));
```

Comment:
I don't think you're suppose to render in `App.js`.  This component should just return the Navbar.  This code goes in index.js.  But I think this is a good way to test out how to use React.